commit b1366224534a5ff48a08dd415290fb8729fab093 (refs/changes/07/2607/2)
Author: Jared Morrow <jm@joyent.com>
Date:   2017-09-19T23:17:07+00:00 (2 years, 1 month ago)
    
    MANTA-3418 manta log uploader lockfile logic removes lockfile when it fails to obtain lock
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/deps/manta-scripts b/deps/manta-scripts
index 4b86dd1..253ae31 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit 4b86dd132d68cb177f066a83d90b8fdc9ac2cde9
+Subproject commit 253ae31df06a8b5a3d520e3874d0eb060dda4fa3
